libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -164,7 +172,11 @@ public void updateLocation(Location location) {
    * @param maxFpsThreshold to be used to limit map frames per second
    */
   public void updateMapFpsThrottle(int maxFpsThreshold) {
-    mapFpsDelegate.updateMaxFpsThreshold(maxFpsThreshold);
+    if (mapFpsDelegate != null) {
+      mapFpsDelegate.updateMaxFpsThreshold(maxFpsThreshold);
```

Comment:
So this is not updating the settings object if the `mapFpsDelegate` is not null, is that by design? See my other comment about separating settings into their respective contexts